src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import calculator.model.Calculator;
+import calculator.service.CalculatorService;
+import calculator.service.CalculatorServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CalculatorTest {
+
+    private Calculator calculator = new Calculator();
+    private CalculatorService calculatorService = new CalculatorServiceImpl();
+
+    @DisplayName("음수를 입력할 경우, IllegalArgumentException이 발생하며 프로그래밍이 종료된다")
+    @Test
+    void invalid_input_negative_then_throw_IllegalArgumentException() throws Exception {
+        assertThrows(IllegalArgumentException.class, () -> {
+            String input = "-1,2:3";
+            List<String> validNumber = calculatorService.findValidNumber(input);
+            calculator.calculatePlusNumber(validNumber);
+        });
+    }
```

Comment:
given/when/then 패턴을 사용하면 테스트 정확도가 더 높을 것 같아요 ! 
지금은 findValidNumber에서 예외가 발생해도 테스트가 성공할 수도 있는 위험이 있어 보여요

Reply:
흠 그러네요 이런 경우에는 모킹을 사용해야할까요?
이번에 제가 궁금했던 부분은 테스트코드를 어디까지 작성해야할까? 도 있는데,

출력문, 즉 Output에 대해 테스트 코드를 작성해야한다고 생각하시나요?

Reply:
@CodingMasterLSW 

```suggestion
    @DisplayName("음수를 입력할 경우, IllegalArgumentException이 발생하며 프로그래밍이 종료된다")
    @Test
    void invalid_input_negative_then_throw_IllegalArgumentException() throws Exception {
        // given
        List<String> validNumber = List.of("-1", "2", "3");
    
        // when & then
        assertThrows(IllegalArgumentException.class, () -> {
            calculator.calculatePlusNumber(validNumber);
        });
    }
```

요런 느낌을 말하려던 거였습니다!  `calculatorService.findValidNumber(input)` 를 꼭 사용하지 않아도 된은 상황으로 보여서요.

출력문에 대한 테스트는 `OutputView` 에 대한 단위 테스트를 말씀하시는 걸까요? 출력 구문이나 포메팅이 많이 이루어진다면 테스트를 하는 것도 좋아 보이긴 합니다! 이미 통합 테스트를 통해 검증을 하고 있다고 해도요.